packages/monitors/src/SyntheticPegMonitor.js

```diff
@@ -142,11 +142,13 @@ class SyntheticPegMonitor {
 
     const volData = await this._checkPricefeedVolatility(pricefeed);
 
-    if (!volData) {
+    if (Object.keys(volData).includes("latestTime")) {
```

Comment:
is this correct? should it not be `!Object.keys(volData).includes("latestTime")`? the way I read this is that it will generate the warning if there _is_ the key `latestTime`. is that intended?

Reply:
scratch that. I see that later down you *only* append the `latestTime` iff there is an error and so this will only be thrown if that key is present. Perhaps you could consider a more descriptive key for this error state, other than `latestTime`?

Reply:
Yeah I can rename it to `errorData` and include `latestTime` in the error data